.travis.yml

```diff
@@ -12,6 +12,8 @@ addons:
       - debian-sid
     packages:
       - shellcheck
+      - pwgen
+      - whois
```

Comment:
In addition to the commit message? Do you think somebody would find it, if necessary, with `git log -G whois`?